src/components/EditorPage/EditorPage.types.ts

```diff
@@ -16,8 +16,10 @@ import {
   ResetCameraAction
 } from 'modules/editor/actions'
 import { openModal, OpenModalAction } from 'modules/modal/actions'
+import { Project } from 'modules/project/types'
 
 export type Props = {
+  project: Project | null
```

Comment:
All the other containers use `currentProject` for the name of this variable (coming from `getCurrentProject(state)`)